source/parser.hera

```diff
@@ -1794,14 +1795,42 @@ FunctionExpression
       signature: {
         modifier: {},
       },
-      children: [ref, " => ", body],
+      children: [parameters, " => ", body],
       ref,
       body,
       ampersandBlock: true,
       block,
       parameters,
     }
 
+  # BinaryOp function shorthand
+  OpenParen:open !Identifier !Not BinaryOp:op CloseParen:close ->
+    const refA = makeRef("a")
+      refB = makeRef("b"),
+      body = [refA, op, refB]
+
+    const parameters = {
+      type: "Parameters",
+      children: [open, refA, ",", refB, close],
```

Comment:
I don't think we should use `open` and `close` here; rather, either the whole expression or body should be wrapped in these parentheses. Otherwise e.g. `(+)+5` will bring the `+5` into the lambda which seems weird (not that this example is especially meaningful, but something like `(+) compose (*)` for a custom `operator compose` might be).